sw/device/tests/BUILD

```diff
@@ -4843,15 +4843,17 @@ opentitan_test(
         "i2c_target_test.c",
     ],
     cw310 = cw310_params(
```

Comment:
Not your fault but the comment above the `opentitan_test` looks obsolete?

Reply:
Good catch, I'll remove it in a follow-up PR. I don't want to trigger CI again. Thanks.